src/pages/companies/[id]/expenses/[expenseId]/edit-expense.tsx

```diff
@@ -160,6 +191,26 @@ const EditExpense = ({
           </select>
         </div>
 
+        <div className={styles.formRow}>
+          <label className={styles.inputLabel} htmlFor="account">Contact</label>
+          <select id="account" name="account"
+            disabled={
+              // Ideally this would be handled in the API to avoid retrieving all of the contacts when a contact is not applicable for this transaction.
+              // However, since for simplicity this demo has the transactions stored locally in the browser, we are doing this check locally.
+              !isSupplierApplicable(expenseTransaction)
+            }>
+            {suppliers!.map((supplier) => (
+              <option
+                key={supplier.id}
+                value={supplier.id}
+                selected={supplier.id === expenseTransaction.contactRef?.id}
```

Comment:
React moans a little when you use `selected` on the option, can you use `value` on the `select` pls?

Reply:
`defaultValue`, even

Reply:
Its the same as the other fields in this page. Should I change those too to be consistent?

Reply:
I've updated those in my pr

Reply:
and that's merged now!